src/utils.js

```diff
@@ -70,3 +71,22 @@ export function pauseEvent(e) {
   e.stopPropagation();
   e.preventDefault();
 }
+
+export function getKeyboardValueMutator(e) {
+  switch (e.keyCode) {
+    case keyCode.UP:
+    case keyCode.RIGHT:
+      return (value, props) => value + props.step;
+
+    case keyCode.DOWN:
+    case keyCode.LEFT:
+      return (value, props) => value - props.step;
+
+    case keyCode.END: return (value, props) => props.max;
+    case keyCode.HOME: return (value, props) => props.min;
+    case keyCode.PAGE_UP: return (value, props) => value + props.step * 2;
```

Comment:
Ok, i understand. But please make sure you update the api doc in readme correctly.